src/components/relativeTime.vue

```diff
@@ -26,6 +26,10 @@ export default {
       values.unshift({ unit: 'h', number: t % 24 })
       t = Math.floor(t / 24)
       values.unshift({ unit: 'd', number: t })
+      t = Math.floor(t / 30)
+      values.unshift({ unit: 'mo.', number: t })
+      t = Math.floor(t / 12)
+      values.unshift({ unit: 'y', number: t })
```

Comment:
Above code consider the time difference as epoch time and convert it into date-time. However what we want is the time which has been spent.

Reply:
```
t = new Date(100000000000) // Sat Mar 03 1973 19:46:40 GMT+1000 (+10)
```
What we wanted is soemthing like `15y 8mo 13 d ....`
What we got is `3y 2mo 3d 19h ...`

Reply:
Sure, instead of `100000000000` I meant to write something like `Date.now() - block.time`, but anyway it won't tread leap years properly, so let's keep as it is for now.

The other option would be is to use `moment.js`, but I'm not sure that is it reasonable.